swim/disseminator_test.go

```diff
@@ -461,7 +461,7 @@ func TestThrottleBidirectionalFullSyncs(t *testing.T) {
 	tnodes := genChannelNodes(t, maxJobs+1)
 
 	bootstrapNodes(t, append(tnodes, tnode)...)
-	waitForConvergence(t, time.Second, append(tnodes, tnode)...)
+	waitForConvergence(t, 100, append(tnodes, tnode)...)
```

Comment:
Can we have units here too? I.e. `100 * time.Millisecond` or whichever unit is the default here?